src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.exception.FruitStoreException;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final String FIRST_LINE = "b,banana,21";
+    private static final String FILE_NAME_NOT_OK = "input_any.csv";
+    private static final String FILE_NAME_OK = "input.csv";
+    private static final String FILE_NAME_EMPTY_NOT_OK = "input_empty.csv";
+    private static ReaderServiceImpl readerService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test(expected = FruitStoreException.class)
+    public void readFile_FileNotFound_notOk() {
+        readerService.readFileToList(FILE_NAME_NOT_OK);
+        fail("Expected " + FruitStoreException.class.getName()
+                + " to be thrown for not existing file, but it wasn't");
+    }
+
+    @Test(expected = FruitStoreException.class)
+    public void readFile_Empty_NotOk() {
+        readerService.readFileToList(FILE_NAME_EMPTY_NOT_OK);
+        fail("Expected " + FruitStoreException.class.getName()
+                + " to be thrown for empty file, but it wasn't");
+    }
+
+    @Test
+    public void readFile_Ok() {
+        List<String> linesFromFile = readerService.readFileToList(FILE_NAME_OK);
+        String expected = FIRST_LINE;
+        String actual = linesFromFile.get(1);
+        assertEquals("First line Expected: " + expected + ", but was: "
+                + actual, expected, actual);
```

Comment:
enougth to assert expected and actual, the message would be provided by JUnit. fix all of it

Reply:
took away